internal/monitors/expvar/config.go

```diff
@@ -57,6 +56,24 @@ type MetricConfig struct {
 	Type string `yaml:"type" validate:"required,oneof=gauge cumulative"`
 	// Metric dimensions
 	DimensionConfigs []DimensionConfig `yaml:"dimensions"`
+	// Metric value path separator character in JSON. The default character is /
+	PathSeparator string `yaml:"pathSeparator" default:"/"`
+	// The PathSeparator string character as rune
+	pathSeparator rune
```

Comment:
Do not treat config objects as scratch pads for caching data or deriving values. This makes things like config comparison when determining which montiors to restart more complicated.